channels/smartcard/client/smartcard_main.c

```diff
@@ -428,7 +428,7 @@ static UINT smartcard_process_irp(SMARTCARD_DEVICE* smartcard, IRP* irp)
 			case SCARD_IOCTL_ISVALIDCONTEXT:
 			case SCARD_IOCTL_CANCEL:
 			case SCARD_IOCTL_ACCESSSTARTEDEVENT:
-			case SCARD_IOCTL_RELEASESTARTEDEVENT:
+			case SCARD_IOCTL_RELEASETARTEDEVENT:
 				asyncIrp = FALSE;
```

Comment:
Are you sure about this renaming ?

Reply:
this way it conforms to `[MS-RDPESC]` naming.